std_msgs/String id
std_msgs/Time timestamp
std_msgs/Bool status
std_msgs/String data